lib/models/device.ts

```diff
@@ -1734,7 +1842,9 @@ const getDeviceModel = function (deps, opts) {
 		 * 	// raspberry-pi
 		 * });
 		 */
-		getDeviceSlug: async (deviceTypeName) => {
+		getDeviceSlug: async (
+			deviceTypeName: string,
+		): Promise<string | undefined> => {
```

Comment:
:grey_exclamation: `getManifestBySlug` throws instead of returning `undefined.
```suggestion
		): Promise<string> => {
```

Reply:
@thgreasi in the catch:
```ts
				if (error instanceof errors.BalenaInvalidDeviceType) {
					return;
				}
¬``

Reply:
Indeed you are right, my bad.